plugins/tileacq.py

```diff
@@ -568,6 +573,45 @@ def _check_fov(self, das, sfov):
             sfov = asfov
         return sfov
 
+    def _memory_check(self, _):
+        """
+        Makes an estimate for the amount of memory that will be consumed during
+        stitching and compares it to the available memory on the computer.
+        Displays a warning if memory exceeds available memory.
+        returns (bool): True (memory is sufficient), False (memory will be exhausted)
+        """
+
+        if self.stitch.value:
+            # Number of pixels for acquisition
+            pxs = 0
+            for s in self._get_acq_streams():
+                if hasattr(s, 'emtResolution'):
+                    pxs += s.emtResolution.value[0] * s.emtResolution.value[1]
+                elif hasattr(s, 'repetition'):
+                    pxs += s.repetition.value[0] * s.repetition.value[1]
+                else:
+                    logging.warning("Resolution of stream %s cannot be determined." % s)
+            pxs = pxs * self.nx.value * self.ny.value
+
+            # Memory calculation
+            mem_est = pxs * 2e-8  # Proportionality factor found empirically.
```

Comment:
Move the factor to a constant, declared above the method.

I'd find it also more readable to have the mem_est in bytes. Then the factor would be in bytes/px. You can convert to Gb in the print statement.